app/src/main/java/org/oppia/app/profile/ProfileChooserViewModel.kt

```diff
@@ -23,6 +24,7 @@ class ProfileChooserViewModel @Inject constructor(
   }
 
   lateinit var adminPin: String
+  lateinit var adminProfileId: ProfileId
```

Comment:
Is ``lateinit`` the correct modifier here? It seems like it's possible from the code below for there to be no admin or  do we want the app to crash in that case?

Reply:
Updated the code to crash when there is no admin. But we should be guaranteed that there is an admin. I kept them as lateinit variables.